members/CAL000316.yaml

```diff
@@ -37,8 +37,8 @@ contact_form:
           required: true
     - javascript:
         - name: accent-replace
-          values: ["$NAME_FIRST", "$NAME_LAST"]
-          selectors: ["#txtFirstName", "#txtLastName"]
+          values: ["$NAME_FIRST", "$NAME_LAST", "$ADDRESS_STREET"]
+          selectors: ["#txtFirstName", "#txtLastName", "#txtAddress"]
           commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[èēéë]/g,'e').replace(/[àåáä]/g, 'a').replace(/[Á]/g, 'A').replace(/[.&,]/g, '').replace(/[ý]/g, 'y').replace(/[íï]/g, 'i').replace(/[óö]/g, 'o').replace(/[ñ]/g, 'n'); }"]
```

Comment:
Do you need to add `.replace(/[Ó]/g, 'O')` to this one too?

Reply:
Not necessary for this one